### PR TITLE
Add 7.9 branches to all applicable books

### PR DESCRIPTION
**Pull request by chriscressman**
A place for technical writers to coordinate the docs configuration changes that should be applied shortly after the 7.9.0 feature freeze.

### CODE REVIEW

**Comment by chriscressman** [comment]
I've added the 7.9 branches to the three Enterprise Search books.

File: conf.yaml [hunk 1]
```diff
@@ -1716,8 +1716,8 @@ contents:
           - title:      Ingest Management Guide
             prefix:     en/ingest-management
             current:    7.8
-            branches:   [ master, 7.x, 7.8 ]
-            live:       [ master, 7.x, 7.8 ]
+            branches:   [ master, 7.x, 7.8, 7.9 ]
+            live:       [ master, 7.x, 7.8, 7.9 ]
```
Comment (bmorelli25) [inline conf.yaml:1720]: @dedemorton, I think the order here changes the order in the dropdown.

```suggestion
            branches:   [ master, 7.x, 7.9, 7.8 ]
            live:       [ master, 7.x, 7.9, 7.8 ]
```

File: conf.yaml [hunk 2]
```diff
@@ -912,7 +912,7 @@ contents:
           - title:      Observability
             prefix:     en/observability
             current:    master
-            branches:   [ master ]
+            branches:   [ master, 7.9 ]
```
Comment (dedemorton) [inline conf.yaml:915]:
```suggestion
            branches:   [ master, 7.x, 7.9 ]
```

Comment (bmorelli25) [reply conf.yaml]: 🔥

File: conf.yaml (hunk 1 above)
Comment (dedemorton) [reply conf.yaml]: good catch!

**Comment by chriscressman** [comment]
@lcawl @gtback I'm taking PTO July 16-17. If this wraps up and is ready to merge during that time, please do so on my behalf. Thanks!

**Comment by lcawl** [comment]
> @lcawl @gtback I'm taking PTO July 16-17. If this wraps up and is ready to merge during that time, please do so on my behalf. Thanks!

Thanks for setting this up, @chriscressman !  I'm just getting logstash and security-docs branches added here, then I'll merge when it's clean

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs rebuild

**Comment by lcawl** [comment]
@elasticmachine, run elasticsearch-ci/docs rebuild

**Comment by lcawl** [comment]
@elasticmachine, run elasticsearch-ci/docs rebuild

**Pull request merged**